### PR TITLE
Add ruleset for Aarhus University.

### PR DESCRIPTION
**Pull request by kimsey0**
Aarhus University in Denmark. In top 10k on Alexa.

Domains found using Sublist3r, automatically filtered to remove for example domains with invalid certificates, then manually reviewed.

### CODE REVIEW

**Comment by kimsey0** [comment]
@Hainish: Shouldn't this be labelled [top-10k](https://www.alexa.com/siteinfo/au.dk) instead?

**Comment by Hainish** [comment]
@kimsey0 we might be using a slightly outdated ranking list.  Looks like it's moving up in popularity.

**Comment by kimsey0** [comment]
@Hainish, @J0WI: fetch.sh fails for `applemdm.auit.au.dk`, `vipomat.hum.au.dk`, `maillist.au.dk` and `vc.au.dk` because of certificate errors.

`applemdm.auit.au.dk` doesn't actually seem to have a valid certificate, so I'll remove that one, but the other three have valid certificates signed by Thawte (intermediate Thawte RSA CA 2018, root DigiCert Global Root CA), Let's Encrypt (intermediate Let's Encrypt Authority X3, root ISRG Root X1) and Thawte, respectively.

The intermediate certificates seem to have been issued after the last time the certificate store for https_everywhere_checker was updated (#6790). Should I reopen #6569 or open a new issue about this?

**Comment by kimsey0** [comment]
@J0WI: I'm also not able to get cURL to accept the certificates, but try in a browser - it works there.

Should I just remove these domain from the ruleset? Both `maillist.au.dk` and `vc.au.dk` contain login forms, so I'd prefer not to.

**Comment by J0WI** [comment]
Firefox caches intermediate certs. `$ firefox --profile $(mktemp -d) --new-instance https://vipomat.hum.au.dk` does also fail.

We use comments to document non-functional hosts:
https://github.com/EFForg/https-everywhere/blob/master/CONTRIBUTING.md#non-working-hosts

**Comment by kimsey0** [comment]
That's doesn't seem to be the problem. Firefox on Windows successfully loads the site for me with a new profile, as does Chrome, Internet Explorer and Edge. Chrome on Android loads it successfully. Lynx on Ubuntu complains about the certificate just like cURL, even after a `sudo update-ca-certificates`.

According to SSL Labs, the certificate chains for [vipomat.hum.au.dk](https://www.ssllabs.com/ssltest/analyze.html?d=vipomat.hum.au.dk), [maillist.aau.dk](https://www.ssllabs.com/ssltest/analyze.html?d=maillist.au.dk) and [vc.au.dk](https://www.ssllabs.com/ssltest/analyze.html?d=vc.au.dk&latest) are incomplete, which probably explains it. I'll disable the domains for now and contact au.dk to have them fix it.

File: src/chrome/content/rules/Au.dk.xml [hunk 1]
```diff
@@ -0,0 +1,875 @@
...
+	<target host="dna-robotics.wiki.au.dk" />
+
+	<!-- CDN -->
+	<target host="www.aucdn.dk" />
+		<test url="http://www.aucdn.dk/2016/assets/img/logos.svg" />
+
+	<rule from="^http:" to="https:" />
+</ruleset>
+
+<!--
```
Comment (J0WI) [inline src/chrome/content/rules/Au.dk.xml]: Please move the comments to the top.

Comment (kimsey0) [reply src/chrome/content/rules/Au.dk.xml]: Done.

**Comment by J0WI** [comment]
Let's just reopen to try again.

**Comment by J0WI** [comment]
All tests passed, thanks!

**Comment by kimsey0** [comment]
Thank you!

**Pull request merged**